crates/rpc/rpc-builder/src/auth.rs

```diff
@@ -397,32 +399,25 @@ impl AuthServerHandle {
         format!("ws://{}", self.local_addr)
     }
 
-    fn bearer(&self) -> String {
-        format!(
-            "Bearer {}",
-            self.secret
-                .encode(&Claims {
-                    iat: (SystemTime::now().duration_since(UNIX_EPOCH).unwrap() +
-                        Duration::from_secs(60))
-                    .as_secs(),
-                    exp: None,
-                })
-                .unwrap()
-        )
-    }
-
     /// Returns a http client connected to the server.
-    pub fn http_client(&self) -> jsonrpsee::http_client::HttpClient {
+    pub fn http_client(&self) -> jsonrpsee::http_client::HttpClient<JwtSecretService<HttpBackend>> {
```

Comment:
public fn signature change. is it fine or better to create a new function?

Reply:
this is fine!